pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPoliciesImpl.java

```diff
@@ -66,7 +66,7 @@ public class OffloadPoliciesImpl implements Serializable, OffloadPolicies {
     public static final int DEFAULT_OFFLOAD_MAX_THREADS = 2;
     public static final int DEFAULT_OFFLOAD_MAX_PREFETCH_ROUNDS = 1;
     public static final ImmutableList<String> DRIVER_NAMES = ImmutableList
-            .of("S3", "aws-s3", "google-cloud-storage", "filesystem", "azureblob", "aliyun-oss");
+      .of("S3", "aws-s3", "google-cloud-storage", "filesystem", "azureblob", "s3-compatible-storage");
```

Comment:
It's better to keep the original style, this change will break some users who already take it.

Reply:
ok. i will fix it for code compatible.